examples/contracts/generic_contract/src/bin/schema.rs

```diff
@@ -7,7 +7,7 @@ fn main() {
 
     write_api! {
```

Comment:
To be considered.

Create new `sv_write_api` which will allow us to omit writing all of these types as those are almost always the same.

Reply:
Should be resolved by https://github.com/CosmWasm/sylvia/issues/362